ansible/roles/host_setup/tasks/pin_kernel.yml

```diff
@@ -18,44 +18,59 @@
     filter: "ansible_kernel"
 
 - name: Check Kernel Version
```

Comment:
the intent of this check was to ensure that the deployed kernel on a given host met our minimum requirements. If we go down the path of pinning to a specific kernel version, we can eliminate this task.